charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -57,6 +57,10 @@ spec:
       topologySpreadConstraints:
         {{- $constraints | toYaml | nindent 8 }}
       {{- end }}
+      securityContext:
```

Comment:
`securityContext:` should be inside the `with` since it only needs to be there when the value is set.